pkg/controller/volume/attachdetach/cache/desired_state_of_world.go

```diff
@@ -382,23 +368,14 @@ func (dsw *desiredStateOfWorld) GetVolumesToAttach() []VolumeToAttach {
 						VolumeName:               volumeName,
 						VolumeSpec:               volumeObj.spec,
 						NodeName:                 nodeName,
-						ScheduledPods:            getPodsFromMap(volumeObj.scheduledPods),
+						ScheduledPods:            volumeObj.scheduledPods,
```

Comment:
This returns unsynchronized map to the caller. A/D controller reconciler will read it and populator will write to it in parallel.